### PR TITLE
Update calendar.yml for Dani Gellis' talk 4/4/18

### PR DESCRIPTION
**Pull request by danisyellis**

### CODE REVIEW

File: models/calendar.yml [hunk 1]
```diff
@@ -304,13 +304,14 @@
 
     - 7:45 PM:
         type: Talk
-        emoji: 🔍
-        title: How to Navigate Large Codebases When You're Used To Simple Projects
-        description: TBD
+        emoji: 🔍💻💥
+        title: Honey, I blew up the code!
+        description: For newer developers who've only worked on personal projects with small codebases, making the jump to working on a company's codebase or an open source project can be daunting. The codebase looks huge!
```
Comment (billyroh) [inline models/calendar.yml]: I believe this you'll need to pipe-escape this! Like so:

```yml
description: |
  For newer developers who've only...
```

File: models/calendar.yml [hunk 2]
```diff
@@ -304,13 +304,14 @@
...
+        In this talk, you'll learn some useful tips for navigating, and contributing to, large codebases.
         person:
           name: Dani Gellis
-          github: danisyellis
-          pronouns: # examples: she/her, they/them, he/him  (bit.ly/pronoun-faq)
+          github/twitter: danisyellis
```
Comment (billyroh) [inline models/calendar.yml]: Unfortunately, our parser only allows for a GitHub handle or a Twitter handle, but not both. So it'll need to be either:

```yml
github: danisyellis
```

Or:

```yml
twitter: danisyellis
```

Whichever you prefer 😄

File: models/calendar.yml (hunk 1 above)
Comment (danisyellis) [reply models/calendar.yml]: Done :-)

File: models/calendar.yml (hunk 2 above)
Comment (danisyellis) [reply models/calendar.yml]: Done. Thanks for letting me know.

**Comment by danisyellis** [comment]
Changes made. Thanks @billyroh

**Comment by billyroh** [comment]
Looks great! Thanks for updating 🙌

**Pull request merged**